.github/workflows/qa.yml

```diff
@@ -10,7 +10,7 @@ jobs:
         runs-on: ubuntu-22.04
         strategy:
             matrix:
-                php: [ '7.2', '7.3', '7.4', '8.0' ]
+                php: [ '7.2', '7.4', '8.0', '8.1' ]
```

Comment:
`"php": "^7.2 || ^8.0"` Dit staat nog in de composer.json, moet dat niet overeenkomstig veranderd worden, of hier nog steeds 7.2 en 7.3 worden toegestaan?

Reply:
Ik vond PHP 7.3 overbodig om te testen. Dit gebruiken we nergens. PHP 8.1 lijkt me wel nuttig 😅